ui/src/app/cohort-search/demographics/demographics.component.tsx

```diff
@@ -427,76 +336,76 @@ export class Demographics extends React.Component<Props, State> {
-                <div ref={(el) => this.ageWrapper = el} id='count-wrapper'>
-                  {calculating
-                    ? <Spinner size={16}/>
-                    : <span style={styles.count} id='age-count'>
-                      {count.toLocaleString()}
-                    </span>
-                  }
-                </div>
-                <Nouislider behaviour='drag'
-                  connect
-                  instanceRef={(slider) => this.onSliderInit(slider)}
-                  onChange={() => this.updateAgeSelection()}
-                  onSlide={(v) => this.onSliderUpdate(v)}
-                  range={{min: +defaultMinAge, max: +defaultMaxAge}}
-                  start={[+defaultMinAge, +defaultMaxAge]}
-                  step={1}/>
-              </div>
-              <div style={{width: '2.5rem'}}>
-                <NumberInput style={styles.ageInput}
-                  min={minAge} max={defaultMaxAge}
-                  value={maxAge}
-                  onBlur={() => this.onMaxBlur()}
-                  onChange={(v) => this.onMaxChange(v)}/>
+            <div style={{...styles.slider, marginBottom: '0.75rem'}}>
+              <div ref={(el) => this.ageWrapper = el} id='count-wrapper'>
+                {calculating
+                  ? <Spinner size={16}/>
+                  : <span style={styles.count} id='age-count'>
+                    {count.toLocaleString()}
+                  </span>
+                }
               </div>
+              <Nouislider behaviour='drag'
+                connect
+                instanceRef={(slider) => this.onSliderInit(slider)}
+                onSlide={(v) => this.onSliderUpdate(v)}
+                range={{min: +defaultMinAge, max: +defaultMaxAge}}
+                start={[+defaultMinAge, +defaultMaxAge]}
+                step={1}/>
             </div>
-            <div style={{marginLeft: '1rem'}}>
-              {ageTypes.map((ageTypeRadio, a) => <div key={a} style={{display: 'inline-block', marginRight: '0.5rem'}}>
-                <input type='radio' name='ageType'
-                  style={{marginRight: '0.25rem'}}
-                  onChange={() => this.onRadioChange(ageTypeRadio.type)}
-                  checked={ageTypeRadio.type === ageType}/>
-                <label>{ageTypeRadio.label}</label>
-              </div>)}
+            <div style={{width: '2.5rem'}}>
+              <NumberInput style={styles.ageInput}
+                min={minAge} max={defaultMaxAge}
+                value={maxAge}
+                onBlur={() => this.onMaxBlur()}
+                onChange={(v) => this.onMaxChange(v)}/>
             </div>
           </div>
-          // List of selectable criteria used for Race, Ethnicity, Gender and Sex
-          : <div style={styles.selectList}>
+          <div style={{marginLeft: '1rem'}}>
+            {ageTypes.map((ageTypeRadio, a) => <div key={a} style={{display: 'inline-block', marginRight: '0.5rem'}}>
+              <input type='radio' name='ageType'
```

Comment:
Can we use RadioButton from input.tsx for consistency?

Reply:
Nice idea, done.